src/ui/common/src/components/integrations/cards/card.tsx

```diff
@@ -27,10 +27,17 @@ import { TruncatedText } from './text';
 
 type IntegrationProps = {
   integration: Integration;
+
+  // Eg: "2 workflows using this integration"
+  numWorkflowsUsingMsg: string;
 };
 
+const paddingRightForNumWorkflowsMsg = 8; // pixels
```

Comment:
@agiron123 is this the right place to handle these spacing constants?

Reply:
Hmm, I think we should remove these and put the values inline.

Reply:
They're not exported or used anywhere else, so these can just go inline.